README.md

```diff
@@ -3854,20 +3854,38 @@ resource cleanup when possible.
 <sup>[[link](#percent-s)]</sup>
 
 * <a name="percent-literal-braces"></a>
-  Prefer `()` as delimiters for all `%` literals, except `%r`. Since parentheses
-  often appear inside regular expressions in many scenarios a less common
-  character like `{` might be a better choice for a delimiter, depending on the
-  regexp's content.
-<sup>[[link](#percent-literal-braces)]</sup>
+  Prefer different kinds of braces for percent literals:
```

Comment:
I don't like this sentence a lot. 

It should be something like "Use the braces that are the most appropriate for the various kinds of percent literals."